dash/dash.py

```diff
@@ -618,7 +618,10 @@ def _collect_and_register_resources(self, resources):
         # add the version number of the package as a query parameter
         # for cache busting
         def _relative_url_path(relative_package_path="", namespace=""):
-            if any(x in relative_package_path for x in ["dcc", "html", "dash_table"]):
+            if any(
+                relative_package_path.startswith(x)
```

Comment:
```suggestion
                relative_package_path.startswith(x + '/')
```
(Or just put the slashes in each list item)

The actual `relative_package_path` values we want to catch here are all like `html/dash_html_components.min.js` right? I'm worried about what would happen if someone made a new package called `html_helpers` or something.

Reply:
oh right, black wants doublequotes 🙄